vs_utils/features/esp.py

```diff
@@ -157,4 +161,28 @@ def calculate_esp(self, mol):
                 grids.append(None)
 
         grids = np.asarray(grids)
+        return charges, radii, grids
+
+    def calculate_esp(self, mol):
+        """
+        Calculate electrostatic potential grid.
+
+        Procedure
+        ---------
+        1. Prepare molecule by adding hydrogens and canonicalizing the
+            orientation and alignment.
+        2. Calculate charges and radii with Antechamber.
+        3. Calculate electrostatic potential grids with PBSA.
+
+        PBSA requires PQR input, which is similar to PDB with charge and
+        radius information added. Neither Antechamber nor PBSA take piped
+        input, so we have to use a lot of temporary files and directories.
+
+        Parameters
+        ----------
+        mol : RDMol
+            Molecule.
+        """
+
+        charges, radii, grids = self.calculate_charges_radii_and_esp(mol)
```

Comment:
Yes, sorry, I thought I closed this pull request, it was superceded by one of the others.